### ISSUE TITLE
3.2.1 seems to interfere with dm_control 

### ISSUE THREAD

**Issue by vmoens** [google-deepmind/mujoco#1883]
TorchRL's CI is failing all of a sudden because of the new minor release:

https://github.com/pytorch/rl/actions/runs/10271094204/job/28420350026?pr=2368

```
  File "/pytorch/rl/env/lib/python3.10/site-packages/dm_control/mujoco/index.py", line 628, in struct_indexer
    attr = getattr(struct, field_name)
AttributeError: 'MjModel' object has no attribute 'tex_rgb'. Did you mean: 'flex_rgba'?
```

`tex_rgb` is also still referenced in the [docs](https://github.com/google-deepmind/mujoco/blob/ebc48d2783da6547f1566d00531d4f202aceeb63/doc/programming/simulation.rst?plain=1#L570)

For now I will just restrict mujoco<3.2.1!

### To reproduce

```bash
# pip3 install dm_control -U
```
then

```python
>>> import dm_control.suite
>>> dm_control.suite.load("cheetah", "run").reset()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/vmoens/venv/rl/lib/python3.11/site-packages/dm_control/suite/__init__.py", line 113, in load
    return build_environment(domain_name, task_name, task_kwargs,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/vmoens/venv/rl/lib/python3.11/site-packages/dm_control/suite/__init__.py", line 148, in build_environment
    env = domain.SUITE[task_name](**task_kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/vmoens/venv/rl/lib/python3.11/site-packages/dm_control/suite/cheetah.py", line 45, in run
    physics = Physics.from_xml_string(*get_model_and_assets())
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/vmoens/venv/rl/lib/python3.11/site-packages/dm_control/mujoco/engine.py", line 443, in from_xml_string
    return cls.from_model(model)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/vmoens/venv/rl/lib/python3.11/site-packages/dm_control/mujoco/engine.py", line 426, in from_model
    return cls(data)
           ^^^^^^^^^
  File "/Users/vmoens/venv/rl/lib/python3.11/site-packages/dm_control/mujoco/engine.py", line 123, in __init__
    self._reload_from_data(data)
  File "/Users/vmoens/venv/rl/lib/python3.11/site-packages/dm_control/mujoco/engine.py", line 407, in _reload_from_data
    model=index.struct_indexer(self.model, 'mjmodel', axis_indexers),
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/vmoens/venv/rl/lib/python3.11/site-packages/dm_control/mujoco/index.py", line 628, in struct_indexer
    attr = getattr(struct, field_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'MjModel' object has no attribute 'tex_rgb'. Did you mean: 'flex_rgba'?
```

**Comment by jonzamora** [comment]
+1, I got the same breaking issue with `mujoco==3.2.1`. Hope this gets patched soon!

**Comment by nimrod-gileadi** [comment]
Apologies. This is fixed by version 1.0.22 of dm-control, which we just pushed. Does that work for you?

**Comment by nimrod-gileadi** [comment]
If you don't want to upgrade to `1.0.22`, you can now restrict to `1.0.21.post1`, which is identical to `1.0.21`, but restricts to `mujoco==3.2.0`.
